pkgs/by-name/zs/zsh-powerlevel10k/package.nix

```diff
@@ -19,6 +20,16 @@ let
       rev = "refs/tags/v${version}";
       hash = "sha256-mVfB3HWjvk4X8bmLEC/U8SKBRytTh/gjjuReqzN5qTk=";
     };
+
+    patches = (oldAtttrs.patches or [ ]) ++ [
```

Comment:
Nit: these parentheses aren't needed, but I suspect they're better for readability anyway.